### PR TITLE
Bitcoin Core 22.0

### PR DESCRIPTION
**Pull request by laanwj**
I hope the site can cope with the new version syntax.

### CODE REVIEW

**Comment by harding** [comment]
Quickly previewed the site on 10498a776b265558f93f244cf8c5a9f85a776a2d .  Home page, release blog post, and release notes all LGTM.  I'm not able to test the download links on a local preview (since I don't have the binaries here), but the test for broken binary links is passing, so it should be ok.

**Comment by harding** [comment]
Whoops, force pushed to trim some excessive copy/paste detail from a previous release.

File: _posts/en/posts/2021-09-13-release-22.0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+---
+Bitcoin Core version 22.0 is now available for [download][download
+page].  See the [release notes][] for more information about the many
+new features and bug fixes in this release.
+
+If have any questions, please stop by the #bitcoin IRC chatroom
+([IRC][irc], [web][web irc]) and we’ll do our best to help you.
+
+[release notes]: /en/releases/22.0/
+[IRC]: irc://irc.freenode.net/bitcoin
```
Comment (flack) [inline _posts/en/posts/2021-09-13-release-22.0.md:26]: that should probably be updated to libera (and the next line, too)

Comment (laanwj) [reply _posts/en/posts/2021-09-13-release-22.0.md]: Updated

**Pull request merged**